rocky/rocky/templates/tasks/partials/task_status.html

```diff
@@ -0,0 +1,15 @@
+{% load i18n %}
+
+{% if task.status.value == "pending" or task.status.value == "queued" %}
+    <i class="icon queued"></i>&nbsp;{{ task.status.value|capfirst }}
+{% elif task.status.value == "running" or task.status.value == "dispatched" %}
+    <i class="icon running"></i>&nbsp;{{ task.status.value|capfirst }}
+{% elif task.status.value == "cancelled" %}
+    <i class="icon cancelled"></i>&nbsp;{{ task.status.value|capfirst }}
+{% elif task.status.value == "failed" %}
+    <i class="icon negative"></i>&nbsp;{{ task.status.value|capfirst }}
+{% elif task.status.value == "completed" %}
+    <i class="icon positive"></i>&nbsp;{{ task.status.value|capfirst }}
+{% else %}
+    <i class="icon neutral"></i>&nbsp;{{ task.status.value|capfirst }}
+{% endif %}
```

Comment:
```suggestion
{% if task.status.value == "pending" or task.status.value == "queued" %}
  {% set status = "queued" %}
{% elif task.status.value == "running" or task.status.value == "dispatched" %}
  {% set status = "running" %}
{% elif task.status.value == "cancelled" %}
  {% set status = "cancelled" %}
{% elif task.status.value == "failed" %}
  {% set status = "negative" %}
{% elif task.status.value == "completed" %}
  {% set status = "positive" %}
{% else %}
  {% set status = "neutral" %}
{% endif %}
<i class="icon {{ status }}"></i>&nbsp;{{ task.status.value|capfirst }}
```
Or better, map these values in the backend?

Reply:
even better, I have removed this partial and just use the value as class. Added these vars in sass